.bundlewatch.config.js

```diff
@@ -14,12 +14,8 @@ module.exports = {
       "maxSize": "180 kB",
     },
     {
-      "path": "client-participation/dist/cached/*/js/polis.js",
-      "maxSize": "150 kB",
-    },
-    {
-      "path": "client-participation/dist/cached/*/js/vis_bundle.js",
-      "maxSize": "200 kB",
+      "path": "client-participation/dist_foo/vis_bundle.js",
```

Comment:
@tamas-soos-toptal, can you tell me about this change?

Reply:
@crkrenn, it was necessary to change the [bundlewatch.yaml](https://github.com/compdemocracy/polis/pull/1341/files/2423d52682e13917ea43d59a59388bd0f2bf3ecb#diff-3f48615a9c0d30dcce674d9ece0476fa80b22a13a825a098adf38f4cbfd2fcdf) file to use webpack for the client-participation container because it uses the config container instead of the polis.config.template.js file. Because of this change, now webpack is responsible for the build process and the existing config of the webpack in client-participation created this dist_foo/vis_bundle.js file by default so it seemed like the best solution to change the bundlewatch.config.js file. I hope this explanation makes sense.

Reply:
Yes, that makes sense. Can you get rid of the "foo"s? Thanks!

Reply:
Yes, that makes sense. Can you get rid of the "foo"s? Thanks!

Reply:
@crkrenn, I changed the dist_foo to dist and as I can see, all the workflows are still running successfully.